app/src/main/java/com/konkuk/hackathon/feature/center/elderregister/ElderRegisterScreen.kt

```
@@ -0,0 +1,677 @@
+        
+        val hour1 = parts1[0].toInt()
+        val minute1 = parts1[1].toInt()
+        val hour2 = parts2[0].toInt()
+        val minute2 = parts2[1].toInt()
+        
+        return when {
+            hour1 < hour2 -> true
+            hour1 > hour2 -> false
+            else -> minute1 < minute2
+        }
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)